lib/poll.js

```diff
@@ -3,8 +3,8 @@ const crypto = require('crypto');
 function Poll() {
   this.id = crypto.randomBytes(20).toString('hex');
   this.voterId = crypto.randomBytes(20).toString('hex');
```

Comment:
We're using `crypto.randomBytes(20).toString('hex')` multiple times. It probably makes sense to extract it out into a helper function.

Something along the lines of:

```js
function generateHash() {
  return crypto.randomBytes(20).toString('hex')
}
```